pom.xml

```diff
@@ -10,12 +10,26 @@
     <version>0.0.1-SNAPSHOT</version>
 
     <properties>
+        <assertj.version>3.15.0</assertj.version>
+        <commons-io.version>2.6</commons-io.version>
         <java.version>1.8</java.version>
         <junit.jupiter.version>5.5.2</junit.jupiter.version>
         <maven.maven-surefire-plugin.version>2.22.2</maven.maven-surefire-plugin.version>
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
```

Comment:
Do we need to pull this dependency in ?
Most of it becomes available by default if we were to upgrade to Java 11.

Reply:
True, but I'm not sure if the goblin in the corner will agree to this upgrade. I'd rather not pull that change in this PR.

Reply:
Hmm agreed. Please create a ticket for that.

Reply:
Done. See here: https://github.com/kwakeroni/GildedRose-Refactoring-Kata/issues/2